Client.Linq.Test/InfluxDBQueryVisitorTest.cs

```diff
@@ -95,6 +95,7 @@ public void ResultOperatorTake()
             Assert.AreEqual(expected, visitor.BuildFluxQuery());
         }
 
+#if NETCOREAPP2_0_OR_GREATER
```

Comment:
Why is there this condition?

Reply:
Because [IQueryable.TakeLast()](https://learn.microsoft.com/en-us/dotnet/api/system.linq.queryable.takelast) used in this test is only available starting with .NET Core 2.0.
So this condition is there to exclude this test when running on non-compatible .NET versions (.NET Framework 4.8 in this case).

Reply:
Make sense, thanks.